spec/features/users/admin_edits_user_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+feature "admin can change user's role", %{
+  As a admin
+  I want to see a list of users
+  So that I can promote/demote them
+} do
+
+  scenario "admin promotes user" do
```

Comment:
there should be a scenario here that tests what happens when a standard user / member attempts to edit another user